fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -572,7 +572,7 @@ private synchronized void receiveInteraction(VehicleSlowDown vehicleSlowDown) th
             );
         }
         bridge.getVehicleControl()
-                .slowDown(vehicleSlowDown.getVehicleId(), vehicleSlowDown.getSpeed(), (int) vehicleSlowDown.getInterval());
+                .slowDown(vehicleSlowDown.getVehicleId(), vehicleSlowDown.getSpeed(), nsToMs((long) (vehicleSlowDown.getInterval() * TIME.SECOND)));
```

Comment:
change field in VehicleSlowDown to long/nanoseonds and you wont need the extra conversion here

Reply:
Is implemented